client/interactions.lua

```diff
@@ -490,8 +490,15 @@ local function getInteractionOptions(interaction)
             local success, result = pcall(canInteract, option, interaction)
 
             if success and result then
-                added += 1
-                currentOptions[added] = option
+                if option.job then
+                    if hasGroup(option.job) then
+                        added += 1
+                        currentOptions[added] = option
+                    end
+                else
+                    added += 1
+                    currentOptions[added] = option
+                end
```

Comment:
```suggestion
                if not option.job or hasGroup(option.job) then
                        added += 1
                        currentOptions[added] = option
                end
```

Reply:
No reason to make a if-else statement jsut follow what I've done previously and use if not option.job or hasGroup(option.job)